app/helpers/ct_table_for/application_helper.rb

```diff
@@ -215,11 +215,19 @@ def class_for_action action, options
 
     def button_for_custom_action record, options, extras
       parsed_extras = parse_extras(extras)
-      if options[:actions][:icons] != false
+      if parsed_extras[:icons] != false
```

Comment:
May `if parsed_extras[:icons]` be enough?

maybe `if parsed_extras[:icons].present?`

Reply:
I updated this logic. It had a bug when `icons` was `"false"`